packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -190,7 +385,9 @@ class FlutterError extends AssertionError {
   /// Include as much detail as possible in the full error message,
   /// including specifics about the state of the app that might be
   /// relevant to debugging the error.
-  FlutterError(String message) : super(message);
+  FlutterError(this._diagnostics) : super(null);
+
+  final List<DiagnosticsNode> _diagnostics;
```

Comment:
does this need to be private? (making it private means it's harder to determine the type of the `FlutterError` constructor argument from the docs)

Reply:
No need to make it private. Updated the constructor for FlutterError to reference this.

Reply:
made public and added documentation.